www/docs/configuration/providers.md

```diff
@@ -125,7 +125,7 @@ providers: [
 |   accessTokenUrl    |               Endpoint to retrieve an access token               |            `string`             |   Yes    |
 |  authorizationUrl   |         Endpoint to request authorization from the user          |            `string`             |   Yes    |
 |      clientId       |                 Client ID of the OAuth provider                  |            `string`             |   Yes    |
-|    clientSecret     |               Client Secret of the OAuth provider                |            `string`             |    No    |
+|    clientSecret     |               Client Secret of the OAuth provider                |            `string`             |   Yes    |
```

Comment:
@balazsorban44 I see there's a mismatch between what's marked as required in `CommonProviderOptions` and [our documentation](https://next-auth.js.org/configuration/providers#oauth-provider-options) ( not just for the `clientId` field but also `profile`, `params`, etc... ) 🧐

I think it might be because what's a required parameter for an "OAuth" provider might not apply equally for the "Email" or "Credentials" providers 💭

Reply:
I discussed this yesterday with @balazsorban44; basically, this table is out of date, and the source of truth is the types on what options are optional for which type of provider. 💆🏽‍♂️

@leeoocca, if you can revert this change for now and keep the ones for the types, I'll be opening a PR soon to fix this part of the documentation 👍🏽